clang/lib/CConv/AVarBoundsInfo.cpp

```diff
@@ -1071,36 +1201,49 @@ bool AVarBoundsInfo::performFlowAnalysis(ProgramInfo *PI) {
       removeBounds(TBK, FlowInferred);
```

Comment:
This is part of `keepHighestPriorityBounds` -- what makes a possible bounds high priority?

Reply:
This is the priority order we use: https://github.com/correctcomputation/checkedc-clang/blob/dd73a007b21dfefa035290954112f169a90383e3/clang/include/clang/CConv/AVarBoundsInfo.h#L83